var/spack/repos/builtin/packages/py-maestrowf/package.py

```diff
@@ -11,10 +11,22 @@ class PyMaestrowf(PythonPackage):
        simulation studies."""
 
     homepage = "https://github.com/LLNL/maestrowf/"
-    url      = "https://github.com/LLNL/maestrowf/archive/v1.1.2.tar.gz"
+    url      = "https://github.com/LLNL/maestrowf/archive/v1.1.4.tar.gz"
+    git      = "https://github.com/LLNL/maestrowf/"
 
     maintainers = ['FrankD412']
 
+    # git branches
+    version('develop', branch='develop')
+    version('master',  branch='master')
+
+    # Pre-release candidates
+    version('1.1.5dev',    sha256='eb3d6f31c233e2cde3b84e15c657002b83ff43d4d6b218b33d023a4f527b9e08')
+    version('1.1.4dev1.0', sha256='67f59eed6fa69fc71b88a0a769de9f080300497d3c30d3a0893eabd0702bc48e')
+    version('1.1.4dev1.1', sha256='c8612b5423b44f11e2a7c4fbc31eb741013245870512ee2dbf7367024517528f')
+
+    # pypi releases
+    version('1.1.6', sha256='27a4ab9072c5b5e2edf91c192d9fe67f040dd45be7f3e44fd9a998ce4cb1e92d')
```

Comment:
```suggestion
    version('1.1.6', sha256='27a4ab9072c5b5e2edf91c192d9fe67f040dd45be7f3e44fd9a998ce4cb1e92d', preferred=True)
```
This will prevent the pre-release candidates from being installed by default.

Reply:
Ah, got it -- thanks for the suggestion. Committing it now.